### DISCUSSION TITLE
Нужны ли нам форумы на гитхабе?

### DISCUSSION THREAD

**Discussion by kana-sama** [ruHaskell/ruhaskell#111] [General]
Вот и увидим, сколько людей тут ответят. Правда вопрос не очень интересный, поэтому на него и отвечать не интересно, поэтому результат может оказаться нерепрезентативным.

**Comment by cblp** [comment]
Да!

**Comment by cblp** [comment]
Можно даже голосовать смайликами 👍 за/👎 против

**Reply by kana-sama** [reply]
Вот это отличная фича, которой очень не хватает в телеге.

**Reply by kana-sama** [reply]
Да еще и ответы к ответам

**Reply by astynax** [reply]
Ответы к ответам не глубже, чем на один уровень вниз, так что настоящих деревьев обсуждений не получить. Но лучше, чем простая портянка.
Так-то пробовали reddit, помнится, но аудитория так и не подтянулась. Может быть с github и получится.

**Comment by sergeyklay** [comment]
Как инструмент сокращения кол-ва тикетов, которые на самом деле и не тикеты вовсе, а так, вопрос, то почему бы и нет :)

**Reply by cblp** [reply]
Мы не страдаем от количества тикетов, нечего сокращать.

**Comment by nixorn** [comment]
Я читаю чат раз в неделю, обычно проматываю, а говорить/спрашивать очень редко есть чего. Какой-то русскоязычный уголок на котором можно увидеть движ данного момента с обсуждаемыми темами был бы гуд.
Но для того чтобы это взлетело надо чтобы была культура прерывания дискуссии в чате и уходить в это место. В чате все смывает, многие ценные мысли уходят просто в ленту и все.

**Reply by cblp** [reply]
если что, у нас уже был Discourse, уже есть [/r/ruhaskell](https://reddit.com/r/ruhaskell)

**Reply by astynax** [reply]
Discourse нужно было поддерживать. Reddit не то чтобы популярен в наших кругах и слишком инаков, а GH всем привычен более-менее.

**Comment by cblp** [comment]
А ещё здесь есть шанс на индексацию поиском

**Comment by cblp** [comment]
Обратная связь из опроса сообщества 2021

> Возможно часть чатов @haskellru и @haskell_learn была бы более полезна в формате Discourse. (В данном случае даже IRC выгоднее, поскольку индексируется поисковиком.) Найти ответы на уже отвеченные вопросы в телеграмм чате сложно. Сейчас выходит, что повторно тратится время и задающими и отвечающими на одни и те же вопросы (хотя, не уверен). Лучше бы из этого росла база знаний (не только за счёт wiki), хотя бы как-то структурировалась. С другой стороны, есть StackOverflow, но у него свои правила. Не уверен, что Gitter для Q&A подходит. Но, видимо, потребность в русском Discourse есть (могу сильно ошибаться). Одновременное обсуждение нескольких тем в случае с чатами сильно затруднено.

Увы, люди не хотят задавать вопросы на форумах, хотят в чатах.

**Reply by dpwiz** [reply]
Люди не хотят задавать вопросы, а хотят получать ответы :smile:
